official/marketplace/pkg/apps/app_yml.go

```diff
@@ -0,0 +1,18 @@
+package apps
+
+import (
+	semver "github.com/blang/semver/v4"
+)
+
+// AppYML is the structure of app.ignite.yml file.
+type AppYML struct {
```

Comment:
```suggestion
// AppConfig is the structure of app.ignite.yml file.
type AppConfig struct {
```

Reply:
We will use the `plugin.AppConfig` from ignite instead of having this here. For now i replaced `ignite/cli` package with latest commit of `main` branch because there's no version with the new app scaffolding changes included. We can merge this PR as it is or remove the `go.mod` replace after release of new version.